poc/mechanism/binary_randomized_response.py

```diff
@@ -0,0 +1,39 @@
+"""
+Binary randomized response mechanism based on Appendix C.1 in
+https://arxiv.org/pdf/2211.10082.pdf.
+"""
+
+import math
+
+from mechanism import DpMechanism
+
+
+class BinaryRandomizedResponse(DpMechanism):
+    DataType = list[int]
+    # Debiasing produces an array of floats.
+    DebiasedDataType = list[float]
+
+    def __init__(self, eps0: float):
+        self.eps0 = eps0
+        self.p = 1.0 / (math.exp(eps0) + 1.0)
+
+    def add_noise(self, data: DataType) -> DataType:
+        raise NotImplementedError()
```

Comment:
Consider pointing to https://github.com/wangshan/draft-wang-ppm-differential-privacy/issues/10 here.

Similar comment on the other mechanisms below.